src/components/Editor/CallSites.js

```diff
@@ -21,6 +21,8 @@ import { isWasm } from "../../utils/wasm";
 
 import actions from "../../actions";
 
+const MAX_COLUMN_BREAKPOINT_COLUMN = 300;
```

Comment:
* This number is arbitrary, based on nothing more than a guess
* We could make this a pref so users could yolo a large number